things/postgres/things.go

```diff
@@ -50,10 +66,10 @@ func (tr thingRepository) Save(ctx context.Context, ths ...things.Thing) ([]thin
 			return []things.Thing{}, err
 		}
 
-		_, err = tx.NamedExecContext(ctx, q, dbth)
-		if err != nil {
+		_, err2 := tx.NamedExecContext(ctx, q, dbth)
```

Comment:
err2? Why? Errors are re-assignable.

Reply:
err and err2 are diffrent types here (generic error and errors.Error) so it can't reassign

Reply:
Here you can probably inline the function and keep `err`. But if you can't do it for other errors then maybe just inline here and find something better than `err2`. Same for others `err2`

Reply:
You're right, I inlined here. There is just one more place where I couldn't do it, because result of function is used out of scope of `if` block. I renamed it from `err2` to `errdb`